community-content/tf_agents_bandits_movie_recommendation_with_kfp_and_vertex_sdk/mlops_pipeline_tf_agents_bandits_movie_recommendation/src/generator/component.yaml

```diff
@@ -39,14 +39,15 @@ outputs:
 - {name: bigquery_table_id, type: String}
 implementation:
   container:
-    image: tensorflow/tensorflow:2.5.0
+    image: python:3.7
```

Comment:
What is the main reason for switching away from the `tensorflow/tensorflow:2.5.0` image? The component seems to still use the tf==2.5.0 package. The common guideline is to try to use an image that contains the most packages you need.